Mechanic2.roboFontExt/lib/mechanic2/ui/formatters.py

```diff
@@ -28,7 +28,8 @@ def attributedStringForObjectValue_withDefaultAttributes_(self, controller, attr
             string.appendAttributedString_(name)
 
             if obj.extensionPrice():
-                attrs[AppKit.NSForegroundColorAttributeName] = AppKit.NSColor.greenColor()
+                priceColor = AppKit.NSColor.colorWithCalibratedRed_green_blue_alpha_(0.3, 0.7, 0, 1)
```

Comment:
I dont like the new green for the prices (its should be in a different PR but is here already so a small review)

I did a quick test with shadows and its already more readable:
![Screen Shot 2019-10-12 at 22 22 33](https://user-images.githubusercontent.com/1190358/66707375-be1ffa00-ed3f-11e9-8846-ed0937e51cea.png)

the patch:
```python
shadow = AppKit.NSShadow.shadow()
shadow.setShadowColor_(AppKit.NSColor.colorWithCalibratedRed_green_blue_alpha_(0, 0, 1, 1))
shadow.setShadowBlurRadius_(6)
shadow.setShadowOffset_((0, -1))
priceColor = AppKit.NSColor.greenColor()
priceAttrs = {
    AppKit.NSFontAttributeName: AppKit.NSFont.systemFontOfSize_weight_(12.0, AppKit.NSFontWeightHeavy),
    AppKit.NSParagraphStyleAttributeName: paragraph,
    AppKit.NSForegroundColorAttributeName: priceColor,
    AppKit.NSShadowAttributeName: shadow,
}
price = AppKit.NSAttributedString.alloc().initWithString_attributes_('\u2003%s' % obj.extensionPrice(), priceAttrs)
string.appendAttributedString_(price)
```

@andyclymer

Reply:
I’m inclined to leave this as it is for now so we can track it with the new UI. I wonder what happens if I mark the conversation as resolved: does your patch go away or can we still refer to it later?